src/slack/views/modals.ts

```diff
@@ -882,3 +882,28 @@ export const publishOktaConnectionModal = async (
     throw error;
   }
 };
+
+export const publishDisconnectConfirmationModal = async (
+  client: WebClient,
+  args: {
+    triggerId: string;
+    connectionName: string;
+    connectionInfoPayload: string;
+  }
+): Promise<void> => {
+  await client.views.open({
+    trigger_id: args.triggerId,
+    view: {
+      ...Surfaces.Modal({
+        title: 'Disconnect?',
```

Comment:
Only make this change if Slack Modal titles will work with all integraiton names. IIRC there's a lmit of 25 characters. Check the docs to confirm and make the changes accordingly
```suggestion
        title: 'Disconnect ${args.connectionName}?',
```

Reply:
Slack Modal Titles have a limit of 24 characters, which is enough for all of the current integration names. I've implemented the change.